cluster-autoscaler/clusterstate/clusterstate.go

```diff
@@ -55,6 +56,11 @@ const (
 
 	// NodeGroupBackoffResetTimeout is the time after last failed scale-up when the backoff duration is reset.
 	NodeGroupBackoffResetTimeout = 3 * time.Hour
+
+	// CloudProviderNodeInstancesCacheRefreshInterval is the interval between cache refreshes.
+	CloudProviderNodeInstancesCacheRefreshInterval = 2 * time.Minute
```

Comment:
CloudProviderNodeInstancesCacheRefreshInterval

Reply:
What needs to be changed

Reply:
I was thinking usign Threshold instead of interval. But ignore this. Not important.

Reply:
Nothing here. I edited it a few times. And got to same version as original one :)